packages/pwa/app/components/image-gallery/index.jsx

```diff
@@ -101,7 +102,19 @@ const ImageGallery = ({imageGroups = [], selectedVariationAttributes = {}, size}
             {heroImage && (
                 <Box {...styles.heroImageGroup}>
                     <AspectRatio {...styles.heroImage} ratio={1}>
-                        <Img alt={heroImage.alt} src={heroImage.disBaseLink} />
+                        <Img
+                            {...getImageProps({
+                                alt: heroImage.alt,
+                                src: heroImage.disBaseLink,
+                                sizes: {
+                                    base: '100vw',
+                                    lg: styles.heroImage.maxWidth[3]
+                                },
```

Comment:
While ProductTile component does not know about the `sizes` and `widths`, the ImageGallery knows the values for them. The difference here is that ImageGallery is more like a bigger component that knows about its layout on the page.